src/olympia/bandwagon/views.py

```diff
@@ -703,14 +704,35 @@ def get_addons_queryset(self):
         return qs.order_by(sort)[:limit]
 
 
+class TranslationAwareOrderingAliasFilter(OrderingAliasFilter):
+    def filter_queryset(self, request, queryset, view):
+        ordering = self.get_ordering(request, queryset, view)
+
+        if len(ordering) > 1:
+            # We can't support multiple orderings easily because of
+            # how order_by_translation works.
```

Comment:
That's a shame.  I would have (naively) thought we can do `order_by_translation` on the `queryset`, pop `name` from `ordering_fields` and then pass the sorted queryset on to `super(...).filter_queryset`.  But I suppose it doesn't matter if frontend doesn't use it.

Reply:
It's a bit problematic because `order_by_translation` uses `.extra()` to hook up the order by and that's very brittle. I tried to find a more generic way by opening up `order_by_translation` and not using `.extra()` but all that left me with wrong joins and duplicate-entries :disappointed: